### PR TITLE
Add image to home

### PR DESCRIPTION
**Pull request by NewsmanWaterpaper**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,6 +7,17 @@
     <div>
         <div>Hello</div>
         <div>Meme</div>
+		
+		<img src="dance.gif">
```
Comment (Kineic) [inline index.html]: Please add width, height and alt attributes.

File: index.html [hunk 2]
```diff
@@ -7,6 +7,17 @@
...
     </div>
     
+    
+    
+    
+    
+    
+    
+
```
Comment (Kineic) [inline index.html]: Please remove the lines selected from the code.

File: index.html (hunk 1 above)
Comment (NewsmanWaterpaper) [reply index.html]: Done

File: index.html (hunk 2 above)
Comment (NewsmanWaterpaper) [reply index.html]: Done.

**Pull request merged**